qcodes/instrument/base.py

```diff
@@ -266,7 +266,7 @@ def print_readable_snapshot(self, update: bool = False,
                 # this may be a multi parameter
                 unit = snapshot['parameters'][par].get('units', None)
             if isinstance(val, floating_types):
-                msg += f'\t{val:.5g} '  # type: ignore[str-format]
+                msg += f'\t{val:.5g} '
```

Comment:
It's not unused but required when you check with the numpy stubs installed. It if was unrequied our CI would have flagged it given the current config.

Reply:
In any case it does not belong in this pr

Reply:
I based that on the [CI failure of the previous commit](https://dev.azure.com/ms-quantum-public/Microsoft%20Quantum%20(public)/_build/results?buildId=15137&view=logs&jobId=347123c7-c6ed-5812-0c14-06a064703b24&j=347123c7-c6ed-5812-0c14-06a064703b24&t=5f8e1bf4-3899-52e0-b0b2-57867ecb13e6):
```
Starting: mypy
==============================================================================
Task         : Bash
Description  : Run a Bash script on macOS, Linux, or Windows
Version      : 3.171.1
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/bash
==============================================================================
Generating script.
========================== Starting Command Output ===========================
/bin/bash --noprofile --norc /home/vsts/work/_temp/41993469-0857-4a4e-9ebb-af6c368b7f56.sh
qcodes/instrument/base.py:269: error: unused 'type: ignore' comment
Found 1 error in 1 file (checked 395 source files)

##[error]Bash exited with code '1'.
Finishing: mypy
```